### DISCUSSION TITLE
Use betamax with httpx

### DISCUSSION THREAD

**Discussion by mielune** [encode/httpx#2149] [Q&A]
Hi community,

I was using betamax in old projects using `requests`, but now I'm using `fastapi` + `httpx` for a full async framework.

I would like to use [betamax](https://betamax.readthedocs.io/en/latest/index.html) to record and replay backend calls done using `httpx` but I don't know how to do this using `httpx` api.
Have you some tips/pointers to help me to achieve this ?

Kind regards,

**Comment by johtso** [comment]
Have you tried [VCR.py](https://github.com/kevin1024/vcrpy)? It seems to support httpx (although it doesn't seem to be listed in the compatibility section of their docs?)

It's listed in the [third party packages section of the httpx docs](https://www.python-httpx.org/third_party_packages/)
